src/main/resources/edu/cornell/kfs/module/ar/document/datadictionary/PaymentApplicationAdjustmentDocument.xml

```diff
@@ -20,29 +20,28 @@
 
 -->
 <beans xmlns="http://www.springframework.org/schema/beans"
-       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p"
-       xsi:schemaLocation="http://www.springframework.org/schema/beans
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:p="http://www.springframework.org/schema/p"
+	xsi:schemaLocation="http://www.springframework.org/schema/beans
                 http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
 
-    <bean abstract="true"
-          id="PaymentApplicationAdjustmentDocument-workflowAttributes-parentBean"
-          parent="WorkflowAttributes">
-        <property name="searchingTypeDefinitions">
-            <list>
-            <!-- KFSPTS-21431, backport FINP-7323, removing 
-		            <ref bean="SearchingType-PaymentApplicationAdjustmentDocument-chartOfAccountsCode-sourceOnly"/> 
-		            <ref bean="SearchingType-PaymentApplicationAdjustmentDocument-accountNumber-sourceOnly"/>
-		        -->
-                <ref bean="SearchingType-AccountsReceivableDocument-createDate-resultsOnly"/>
-                <ref bean="SearchingType-AccountsReceivableDocument-initiatorId-resultsOnly"/>
-                <ref bean="SearchingType-AccountsReceivableDocument-documentStatus-resultsOnly"/>
-                <ref bean="SearchingType-AccountsReceivableDocument-processingChartOfAccountsCode-criteriaOnly"/>
-                <ref bean="SearchingType-AccountsReceivableDocument-processingOrganizationCode-criteriaOnly"/>
-                <ref bean="SearchingType-PaymentApplicationAdjustmentDocument-billingChartOfAccountsCode-criteriaOnly"/>
-                <ref bean="SearchingType-PaymentApplicationAdjustmentDocument-billingOrganizationCode-criteriaOnly"/>
-                <ref bean="SearchingType-PaymentApplicationAdjustmentDocument-financialDocumentReferenceInvoiceNumber-criteriaOnly"/>
-            </list>
-        </property>
-    </bean>
+	<bean abstract="true" id="PaymentApplicationAdjustmentDocument-workflowAttributes-parentBean" parent="WorkflowAttributes">
+		<property name="searchingTypeDefinitions">
+			<list>
+				<!-- KFSPTS-21431, backport FINP-7323, removing 
```

Comment:
formatting is a bit off. These were removed from after billingOrganizationCode-criteriaOnly, placing the comment there would make it more obvious.